ccmlib/scylla_node.py

```diff
@@ -180,7 +180,9 @@ def _start_jmx(self, data):
         args = [jmx_java_bin,
                 '-Dapiaddress=%s' % data['listen_address'],
                 '-Djavax.management.builder.initial=com.scylladb.jmx.utils.APIBuilder',
+                '-Djava.rmi.server.hostname=%s' % data['listen_address'],
```

Comment:
Sure we need both configurations ?

Reply:
I didn't try without this one.
The other way around didn't work
and it seems weird to provide the host address one way and the port in another way.

Reply:
Yeah, both are needed.
Testing with `pytest -s -vv --cassandra-dir=$CASSANDRA_DIR -n 8 nodetool_additional_test.py`

Reply:
o.k., today won't be the day I understand java configuration logic.